app/assets/stylesheets/overlays/_paper_submit_overlay.scss

```diff
@@ -1,11 +1,59 @@
 .paper-submit-overlay {
   text-align: center;
 
-  h1 {
-    margin-bottom: 0.5em;
+  .overlay-title-text-submit h1 {
+      padding-top: 20px;
```

Comment:
Line should be indented 4 spaces, but was indented 6 spaces<br>Properties should be ordered padding-bottom, padding-top